services/api/src/lib/Schema.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require('mongoose');
```

Comment:
I think this wrapper deserves a unit test

Reply:
just added